languages/python/exercises/concept/strings/test_loglines.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import loglines
```

Comment:
Rather than requiring fully qualified names in the tests use `from loglines` to import the appropriate functions. In the long run we're probably going to want to extract data into a canonical representation for our track and re-purpose the existing test generator, so we may as well use the idioms we developed there.

Reply:
Thanks, I will have to take a closer look at those.